initials.c

```diff
@@ -0,0 +1,11 @@
+/* Matthew Kellerman */
```

Comment:
Nicely done! My only suggestion is to include a description of your program in your comment!